export const invokeError = (error) => {
    throw new Error(error)
};
